packages/eas-cli/src/commands/build/download.ts

```diff
@@ -29,6 +29,7 @@ export default class Download extends EasCommand {
     }),
     'dev-client': Flags.boolean({
```

Comment:
According to types, `dev-client` flag is `boolean` (meaning no `undefined`). Is this true? Do we consciously default to `--dev-client`?

Reply:
Sorry but, according to which types? I changed this mostly because the value I get when not passing `--dev-client`
is `undefined`. This basically allows us to 3 different queries, all builds, dev-client builds only and non-dev-client build only  

<img width="937" alt="image" src="https://github.com/user-attachments/assets/c8504d44-31c7-49c0-9890-0f6690e4e64f" />

Reply:
Cool, what you pasted is what I would expect. I guess the types are wrong.

<img width="506" alt="Zrzut ekranu 2025-04-11 o 15 18 46" src="https://github.com/user-attachments/assets/19d7b141-ce12-408e-a3f9-b8dc8fd16c28" />